src/app/components/demo-components.service.ts

```diff
@@ -81,6 +89,13 @@ export class SkyDemoComponentsService {
         url: '/components/confirm',
         getCodeFiles: () => this.getDemoFiles('SkyConfirmDemoComponent')
       },
+      {
+        name: 'Copy to clipboard',
+        icon: 'clipboard',
+        summary: 'The copy to clipboard component allows users to copy sections of content to the clipboard.',
```

Comment:
I'd try to make this match the opening sentence of the overview paragraph in the docs: "The copy to clipboard component creates a button that enables users to copy content to the clipboard."